src/runtime-rs/config/configuration-qemu.toml.in

```diff
@@ -1,5 +1,4 @@
-# Copyright (c) 2017-2019 Intel Corporation
-# Copyright (c) 2021 Adobe Inc.
+# Copyright (c) 2024 IBM Inc.
```

Comment:
IANAL but shouldn't the previous copyrights be retained?

Reply:
Yep, I made the original ones retained and simply added a new one. Thanks!